graphql_compiler/tests/test_helpers.py

```diff
@@ -193,6 +192,10 @@
       _x_count: Int
       uuid: ID
     }
+
+    interface V {
```

Comment:
I still don't think we want to expose `V`. We were hiding it because it's meaningless (there's nothing you can do with it, since `_x_count` is a synthetic field that's only usable from a `@fold`, which requires an edge to it, of which there are none). We weren't hiding it just because it wasn't representable.

Reply:
How about if it has no properties I hide it. (This would be in get_graphql_schema_from_schema graph). I would also add the comment that we hide it because it's meaningless.